x-pack/plugins/observability/scripts/e2e.js

```diff
@@ -6,9 +6,4 @@
  */
 
 /* eslint-disable no-console */
-const { executeSyntheticsRunner } = require('@kbn/synthetics-plugin/scripts/base_e2e');
-const path = require('path');
-
-const e2eDir = path.join(__dirname, '../e2e');
-
-executeSyntheticsRunner(e2eDir);
+console.log('Disabled.');
```

Comment:
why not remove it?

Reply:
I will in a following PR. But removing this now means removing files owned by Ops, which will slow down this PR.